docs/modules/operation/pages/thresholds/create-thresh.adoc

```diff
@@ -0,0 +1,148 @@
+A custom UEI for each threshold makes it easier to xref:thresholds/notification.adoc#threshold-notification[create notifications].
+If left blank, it defaults to the standard thresholds UEIs.
+| leave blank
+
+| Re-armed UEI
+| A custom unique event identifier (UEI) sent into the events system when the threshold is re-armed.
+ | leave blank
+
+|===
+
+[[threshold-test]]
+== Test the threshold
+
+To test the threshold we just created, log a second person into the node you are monitoring.
+Navigate to the *Events* page.
+You should see an event that indicates your threshold triggered when more than one user logged in.
+
+Log out the second user.
+The *Events* page should indicate that the system has re-armed.
+
+[[thresh-cpu]]
+== Create a threshold for CPU use
+This procedure describes how to create an expression-based threshold when the five-minute CPU load average metric reaches or goes above 70% for two consecutive measurement intervals.
+Expression-based thresholds are useful when you need to threshold on a percentage, not the actual value of the data collected.
+
+NOTE: Expression-based thresholds work only if the data sources in question are in the same directory.
+
+. Click the gear icon in the top-right.
+. Under *Performance Measurement*, choose *Configure Thresholds*.
+. Click *Edit* beside the *netsnmp* group.
+. Click *Create New Expression-based Threshold*.
+. Fill in the following information:
+
++
+
+[options="header"]
+[cols="1,3,3"]
+|===
+
+| Field
+| Description
+| Default
+
+| Type
+| Triggers an event when the data source value equals or exceeds the threshold value, and re-arms when it equals or drops below the re-arm value.
+| high
+
+| Expression
+| Divides the five-minute CPU load average by 100 (to obtain the effective load average\*), which is then divided by the number of CPUs.
+This value is then multiplied by 100 to provide a percentage.
+
+(* SNMP does not report in decimals, which is why the expression divides the loadavg5 by 100.)
+| ((loadavg5 / 100) / CpuNumCpus) * 100
+
+| Datasource type
+| The type of data source from which you are collecting data.
+| node
+
+| Datasource label
+| Optional text label.
+Not required for this tutorial.
+| leave blank
+
+| Value
+| Trigger an event when the five-minute CPU load average goes above 70%.
+| 70
+
+| Re-arm
+| Re-arm the system when the five-minute CPU load average drops below 50%.
+| 50
+
+| Trigger
+| The number of consecutive times the threshold value can occur before the system triggers an event.
+In this case, when the five-minute CPU load average goes above 70% for two consecutive polling periods.
+| 2
+
+| Description
+| Optional text to describe your threshold.
+| Trigger an alert when the five-minute CPU load average metric reaches or goes above 70% for two consecutive measurement intervals.
+
+| Triggered UEI
+| See the table in xref:threshold-create[Create a threshold] for details.
```

Comment:
The cross reference link is broken, and wording confusing. Just copying text here from above.
```suggestion
| A custom unique event identifier (UEI) sent into the events system when the threshold is triggered.
A custom UEI for each threshold makes it easier to xref:thresholds/notification.adoc#threshold-notification[create notifications].
If left blank, it defaults to the standard thresholds UEIs.
```